README.md

```diff
@@ -1,33 +1,48 @@
 [![Steam donate](https://img.shields.io/badge/Donate%20via-Steam-00adee.svg?style=flat-square&logo=steam)](https://steamcommunity.com/tradeoffer/new/?partner=85845165&token=M9cQHh8N)
 [![Join the Discord chat](https://img.shields.io/badge/Discord-%23comtress--client-7289da.svg?style=flat-square&logo=discord)](https://discord.gg/CuPb2zV)
 
 
-# Team Comtress 2
+# Welcome to Team Comtress 2!
+Before sharing your ideas and feedback, it's important to have a good understanding of what this project is.
+### What *is* TC2?
+Team Comtress 2 is a fork of an older version of Team Fortress 2, with a community developer team fixing bugs, improving performance, and adding quality of life features, with the goal of having those changes pulled upstream by Valve to the modern game.
+
+### What is TC2 *not*?
+* TC2 is *not* a "pro mod"
+
+While much of the work in TC2 will improve the competitive experience, TC2 does not touch gameplay, balance, or competitive rules.
+
+* A vision for the "good old days" of TF2
+
+TC2 will not feature game design changes, gameplay additions, or almost any form of new content. If you're looking for this kind of content, [Team Fortress 2 Classic](https://tf2classic.com/) is more up your alley.
+
+* The modern game, but with new content/updates
 
-Team Fortress 2, but with a lot of fixes, quality of life improvements and performance optimizations!
+You're looking for [Creators.TF](https://creators.tf/).
 
-## About
+* An alternative, more stable game experience
 
-What is Team Comtress 2? It's a version of Team Fortress 2, based on the recent leak which aims to fix many bugs, performance issues, etc. Imagine it like [mastercomfig](https://mastercomfig.com/) super charged!
+TC2 is based on the 2017 TF2 source code leak, which contained an in-development version of Jungle Inferno. This build is rife with issues and incomplete features, and *is not compatible with* the modern game. It isn't even fully playable on its own.
 
-Obviously, as a leaked build, it's not useful for getting better performance in Casual on its own (you can't use this build to connect to any existing servers), but it can help me a lot if you all can test it, so that I am more confident in sending many of these changes to Valve for them to include in the base game! Please let me know how it works for you!
+## Installation
 
-## Install
+**Windows-only for now** (see [#2](https://github.com/mastercomfig/team-comtress-2/issues/2))
 
-1. [Download](https://github.com/mastercomfig/team-comtress-2/releases/latest) `game_clean.zip` from the latest release.
-2. Copy your current `Team Fortress 2` installation to a new folder.
-3. Extract the ZIP download to this new folder.
-4. Make sure you don't have any configs installed.
+1. Copy your current `Team Fortress 2` installation to a new folder.
+2. In your cloned TF2 folder, delete the `tf/custom` folder, as well as `tf/cfg/autoexec.cfg` and `tf/cfg/config.cfg`.

```

Comment:
I think the word should be "copied" to keep consistency.
"In the TF2 folder that you just copied" or something like that.
Also note that some people may have class configs or different names of configs. No idea how to reword that tho.

Reply:
will do